frontend/src/utils/constants.js

```diff
@@ -4,3 +4,8 @@ export const PASSWORD_STRONG =
 export const PASSWORD_MEDIUM =
   /((?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[^A-Za-z0-9])(?=.{6,}))|((?=.*[a-z])(?=.*[A-Z])(?=.*[^A-Za-z0-9])(?=.{8,}))/;
 export const LETTERS_ONLY = /^[a-zA-Z]+$/;
+export const SIMPLE_NOTIFICATION_TYPES = {
```

Comment:
Rename `SIMPLE_NOTIFICATION_TYPES` to `NOTIFICATION_SEVERITY` and remove it to `/utils/notifications.js`.